docs/lib/auth/fragments/js/advanced.md

```diff
@@ -312,6 +312,10 @@ const { token } = federatedInfo;
 
 By default, Amplify will automatically refresh the tokens for Google and Facebook, so that your AWS credentials will be valid at all times. But if you are using another federated provider, you will need to provide your own token refresh method:
 
+<amplify-callout>
+Note: Automatic token refresh is not supported in React Native.
```

Comment:
To make it clear: Automatic token refresh for Google and Facebook is not supported in React Native. Automatic token refresh is supported when used with Cognito User pool.